src/main/java/uz/anas/gymcrm/dao/TraineeDAO.java

```diff
@@ -53,6 +83,11 @@ public boolean existsByUsername(String username) {
     }
 
     public List<Trainee> findAll() {
-        return new ArrayList<>(trainees.values());
+        try {
+            return new ArrayList<>(trainees.values());
+        } catch (NullPointerException e) {
```

Comment:
never don't catch nullPointerException. If null is expected somewhere use Optional class and correct handling null values

Reply:
Got it, thanks for pointing out mistake. I'll update it for using `Optional` instead of using `NullPointerException` with try catch blocks